_includes/body-landing.html

```diff
@@ -0,0 +1,844 @@
+<body ng-app="Docker" ng-controller="DockerController" id="landing">
+  <nav class="navbar header">
+    <div class="container">
+      <div class="navbar-header">
+        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#header-content" aria-expanded="false">
+          <span class="sr-only">Toggle navigation</span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+        </button>
+        <a class="navbar-brand" href="/">
+          <img
+            src="../images/docker-docs-logo.svg"
+            alt="Docker Docs"
+            title="Docker Docs"
+          />
+        </a>
+      </div>
+      <div class="collapse navbar-collapse" id="header-content">
+        <ul class="nav navbar-nav">
+          <li class="active">
+            <a href="https://docs.docker.com/">Home</a>
```

Comment:
Can we change the links to not include the domain name?

```suggestion
            <a href="/">Home</a>
```

That way it's possible to test the links locally (or on preview domains)